src/content/docs/concepts/schema.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Schema
+---
+
+A Membrane program's graph is defined by its _schema_, autogenerated in `your-program/memconfig.json` based on the schema you configure in the Explorer panel. The schema is a set of types, which can be _fields_, _actions_, or _events_.
```

Comment:
> Also, not sure if the </> will be interpreted as html

Markdown interprets it as plaintext since it's within inline code backticks.

Reply:
The concept of a "member" confuses me (not sure if it might confuse other new users). I edited to this for now, which we can always change:
```
A Membrane program's graph is defined by its _schema_, autogenerated in `<your-program>/memconfig.json` based on the schema you configure in the Explorer panel.

The schema is a set of types that represent nodes in the program's graph. Nodes can be _fields_, _actions_, or _events_.
```